clang/unittests/Format/FormatTest.cpp

```diff
@@ -22537,10 +22537,12 @@ TEST_F(FormatTest, FormatsLambdas) {
                "  }\n"
                "}",
                Style);
-  verifyFormat("std::sort(v.begin(), v.end(),\n"
-               "          [](const auto &foo, const auto &bar) {\n"
-               "  return foo.baz < bar.baz;\n"
-               "});",
+  verifyFormat("void test() {\n"
```

Comment:
beyonce rule please don't change a test to fit your code change, or we get regressions... was the test wrong before?

Reply:
I've answered the equivalent question here: https://github.com/llvm/llvm-project/pull/66755#discussion_r1331456960